Client/Api.php

```diff
@@ -150,4 +152,58 @@ static function ($hit) {
 
         return new Result($hits);
     }
+
+    /**
+     * @throws ConnectException
+     */
+    public function getRecommendedProducts(RecommendationQuery $query, string $debug = ''): array
+    {
+        $this->sanitizeLanguage($query);
+
+        $url = "$this->baseUrl/recommendation/public";
```

Comment:
For better readability: Please use curly braces, if you put a variable inside a string.

```suggestion
        $url = "{$this->baseUrl}/recommendation/public";
```

Reply:
fixed